bundle/Resources/config/services.yaml

```diff
@@ -0,0 +1,87 @@
+parameters:
+    netgen_ibexa_scheduled_visibility.enabled: false
```

Comment:
I would separate this parameters into a different `default_settings.yaml` file.

Reply:
fixed in e155d4bc22522b90a2afca493b7158bbee435ccb